pages/index.tsx

```diff
@@ -1,15 +1,31 @@
-import { useEffect, useState } from 'react';
-
 import Trending from '../components/trending/Trending';
 import request from '../modules/request';
 import { AppWrapper, GlobalStyle } from '../styles/index.styles';
 
-const Home = (): JSX.Element => {
-  const [data, setData] = useState([]);
+interface Community {
+  title: string;
+  url: string;
+  members: number;
+  image: string;
+}
+
+interface HomeProps {
+  communities: Community[];
+}
+const Home = ({ communities }: HomeProps): JSX.Element => {
```

Comment:
Can we plz add a newline above the component to improve readability.

Reply:
Done!